tests/factories/_base.py

```diff
@@ -60,3 +66,69 @@ def build_batch(cls: Generic[T], size: int, **kwargs: Any) -> list[T]:
     @classmethod
     def create_batch(cls: Generic[T], size: int, **kwargs: Any) -> list[T]:
         return super().create_batch(size, **kwargs)
+
+
+class NullableSubFactory(SubFactory, Generic[TModel]):
+    """
+    A SubFactory where te default value can be None.
```

Comment:
`te` -> `the`

Reply:
Done. Also improved te documentation for all Factory helpers in general.